service/telemetry/config.go

```diff
@@ -121,7 +111,7 @@ type MetricsConfig struct {
 
 	// Readers allow configuration of metric readers to emit metrics to
 	// any number of supported backends.
-	Readers []MetricReader `mapstructure:"metric_readers"`
+	Readers MeterProviderJsonMetricReaders `mapstructure:"metric_readers"`
```

Comment:
Maybe too late/this comes from the Configuration WG, but it seems a bit weird to me to have `metrics::metric_readers` instead of just `metrics::readers`

Reply:
Filed open-telemetry/opentelemetry-configuration/issues/20